### PR TITLE
Test @aminya/electron link

### PR DESCRIPTION
**Pull request by aminya**
This is a test for my electron-link pull request. I want to check the Azure pipeline to see if Atom builds successfully.
https://github.com/atom/electron-link/pull/26

### CODE REVIEW

**Comment by aminya** [comment]
My pull request to electron-link is now battle-tested. As you see in the Azure pipe-line this successfully builds Atom without any issues:
https://github.visualstudio.com/Atom/_build/results?buildId=74130&view=results


Can we merge https://github.com/atom/electron-link/pull/26 now?

File: script/package-lock.json [hunk 1]
```diff
@@ -8,6 +8,202 @@
       "resolved": "https://registry.npmjs.org/7zip-bin/-/7zip-bin-4.0.2.tgz",
       "integrity": "sha512-XtGk+IF57pr852UK1AhQJXqmm1WmSgS5uISL+LPs0z/iAxXouMvdlLJrHPeukP6gd7yR2rDTMSMkHNODgwIq7A=="
```
Comment (Tinup-saku) [inline script/package-lock.json:9]: https://status.circleci.com/

File: script/lib/generate-startup-snapshot.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const childProcess = require('child_process');
 const fs = require('fs');
 const path = require('path');
-const electronLink = require('electron-link');
```
Comment (Tinup-saku) [inline script/lib/generate-startup-snapshot.js:4]: https://status.circleci.com/

Comment (Tinup-saku) [reply script/lib/generate-startup-snapshot.js:4]: https://status.circleci.com/

Comment (Tinup-saku) [inline script/lib/generate-startup-snapshot.js:4]: https://status.circleci.com/@khem

Comment (Tinup-saku) [inline script/lib/generate-startup-snapshot.js:4]: https://status.circleci.com/

File: script/package-lock.json (hunk 1 above)
Comment (aminya) [reply script/package-lock.json:9]: What is this spam? I reported this to GitHub.

**Comment by lkashef** [comment]
Hey @aminya, thanks for the contribution and the proactivity to avoid issues upgrading to electron 6.

I would like to review your branch and include it in our electron upgrade branch but it would be best to open a PR on `electron-link`, feel free to mention me directly to get that reviewed and merged in

**Comment by aminya** [comment]
I am going to close this since https://github.com/atom/electron-link/pull/26 is merged now.

**Comment by lkashef** [comment]
@aminya sounds good, I am working on doing a new release for `electron-link` to propagate it to `atom/atom`

**Pull request closed without merging**